### DISCUSSION TITLE
v1.1.8

### DISCUSSION THREAD

**Discussion by dirkgroot** [avisi-cloud/structurizr-site-generatr#408] [Announcements]
### Features

* Identify external software systems by custom tag by @jp7677 in https://github.com/avisi-cloud/structurizr-site-generatr/pull/356
* Make PlantUML exporter configurable  by @jp7677 in https://github.com/avisi-cloud/structurizr-site-generatr/pull/406

### Bugfixes

* Fix broken lunr stemmer link and remove three-character limit for searching by @jp7677 in https://github.com/avisi-cloud/structurizr-site-generatr/pull/390
* Improve tab styling by @jp7677 in https://github.com/avisi-cloud/structurizr-site-generatr/pull/405

### Dependency updates

* Update Structurize-Export to 1.17.0 And Fix Tests by @qtzar in https://github.com/avisi-cloud/structurizr-site-generatr/pull/355
* Update dependency com.structurizr:structurizr-dsl to v1.33.0 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/353
* Update dependency mermaid to v10.6.0 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/341
* Update dependency org.eclipse.jetty:jetty-server to v12.0.3 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/358
* Update dependency org.eclipse.jetty.websocket:jetty-websocket-jetty-server to v12.0.3 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/357
* Update plugin org.jetbrains.kotlin.jvm to v1.9.20 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/362
* Update plugin org.jetbrains.kotlin.plugin.serialization to v1.9.20 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/360
* Update dependency org.jetbrains.kotlin:kotlin-scripting-jsr223 to v1.9.20 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/359
* Update dependency org.jruby:jruby-core to v9.4.5.0 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/364
* Update eclipse-temurin Docker tag to v21.0.1_12-jre-jammy by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/361
* Update dependency mermaid to v10.6.1 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/366
* Update dependency org.jetbrains.kotlinx:kotlinx-serialization-json to v1.6.1 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/369
* Update dependency com.structurizr:structurizr-core to v1.28.0 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/370
* Update dependency org.jetbrains.kotlin:kotlin-scripting-jsr223 to v1.9.21 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/372
* Update plugin org.jetbrains.kotlin.jvm to v1.9.21 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/373
* Update dependency com.structurizr:structurizr-dsl to v1.34.0 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/371
* Update plugin org.jetbrains.kotlin.plugin.serialization to v1.9.21 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/374
* Update gradle/gradle-build-action action to v2.10.0 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/377
* Update dependency org.jsoup:jsoup to v1.17.1 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/376
* Update dependency gradle to v8.5 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/378
* Update actions/setup-java action to v4 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/379
* Update dependency org.jetbrains.kotlinx:kotlinx-html-jvm to v0.10.1 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/380
* Update dependency org.eclipse.jetty.websocket:jetty-websocket-jetty-server to v12.0.4 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/383
* Update dependency com.willowtreeapps.assertk:assertk-jvm to v0.28.0 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/385
* Update dependency com.structurizr:structurizr-core to v1.29.0 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/387
* Update dependency org.eclipse.jetty:jetty-server to v12.0.5 - autoclosed by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/384
* Update dependency net.sourceforge.plantuml:plantuml to v1.2023.13 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/388
* Update dependency org.asciidoctor:asciidoctorj to v2.5.11 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/391
* Update dependency org.codehaus.groovy:groovy-jsr223 to v3.0.20 - autoclosed by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/392
* Update dependency org.eclipse.jetty.websocket:jetty-websocket-jetty-server to v12.0.5 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/393
* Update dependency org.jetbrains.kotlin:kotlin-scripting-jsr223 to v1.9.22 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/394
* Update dependency org.eclipse.jgit:org.eclipse.jgit to v6.8.0.202311291450-r by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/386
* Update dependency org.jetbrains.kotlinx:kotlinx-serialization-json to v1.6.2 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/381
* Update dependency com.structurizr:structurizr-export to v1.18.0 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/375
* Update dependency org.jsoup:jsoup to v1.17.2 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/395
* Update dependency org.slf4j:slf4j-simple to v2.0.10 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/396
* Update plugin org.jetbrains.kotlin.plugin.serialization to v1.9.22 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/398
* Update plugin org.jetbrains.kotlin.jvm to v1.9.22 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/397
* Update gradle/gradle-build-action action to v2.11.1 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/401
* Update actions/upload-artifact action to v4 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/403
* Update actions/download-artifact action to v4 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/402
* Update dependency com.structurizr:structurizr-dsl to v1.35.0 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/399
* Update dependency com.structurizr:structurizr-export to v1.19.0 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/400
* Update dependency org.slf4j:slf4j-simple to v2.0.11 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/407

### Miscellaneous changes

* Fix README formatting by @jp7677 in https://github.com/avisi-cloud/structurizr-site-generatr/pull/365
* Fix rule name by @jp7677 in https://github.com/avisi-cloud/structurizr-site-generatr/pull/368

**Full Changelog**: https://github.com/avisi-cloud/structurizr-site-generatr/compare/1.1.7...1.1.8

<hr /><em>This discussion was created from the release <a href='https://github.com/avisi-cloud/structurizr-site-generatr/releases/tag/1.1.8'>v1.1.8</a>.</em>

**Comment by qtzar** [comment]
Could the Homebrew version be updated please.

**Reply by dirkgroot** [reply]
Ah, thanks for reminding me, totally forgot. It's updated now.
